src/components/List/ListItem.module.scss

```diff
@@ -0,0 +1,112 @@
+.wrapper {
+	list-style: none;
+    padding: 0 0 50px 0;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	position: relative;
+	width: 100%;
+}
+
+.subWrapper {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	place-content: center flex-start;
+	position: relative;
+	min-width: 30%;
+	border-style: solid;
+	border-color: rgb(0, 83, 135);
+	background: rgb(0, 83, 135);
+	color: white;
+	width: 30%;
+
+	@media (max-width: 600px) {
+		width: 100%;
+	}
+}
+
+.subWrapperCompleted {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	place-content: center flex-start;
+	position: relative;
+	min-width: 30%;
+	width: 30%;
+	background-color: rgba(53, 177, 134, 0.5);
+	
+	@media (max-width: 600px) {
+		width: 100%;
+	}
+}
+
+.checkbox {
+	position: absolute;
+	align-items: right;
+	margin-left: 30px;
+	width: 60px;
+	height: 50px;
+	right: 20px;
+}
+
+.remove {
+	position: absolute;
+	right: 0;
+	top: 9px;
+	width: 30px;
+	height: 40px;
+	color:rgba(200, 235, 223, 0.5);
+}
+
+.infoWrapper {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	place-content: center flex-start;
+
+	.info {
```

Comment:
Avoid nesting too much. Use unique classnames instead and assign a classname to every component so it makes debugging easier.